documentation/modules/deploying/proc-deploy-migrate-kraft.adoc

```diff
@@ -0,0 +1,253 @@
+// Module included in the following assemblies:
+//
+// deploying/deploying.adoc
+
+[id='proc-deploy-migrate-kraft-{context}']
+= Migrating to KRaft mode
+
+[role="_abstract"]
+If you are using ZooKeeper for cluster management of your Kafka cluster, you can migrate to using Kafka in KRaft mode. 
+KRaft mode replaces ZooKeeper for distributed coordination, offering enhanced reliability, scalability, and throughput.
+
+During the migration, you install a quorum of controller nodes as a node pool, which replaces ZooKeeper for management of your cluster. 
+You enable KRaft migration in the cluster configuration by applying the `strimzi.io/kraft: migration` annotation.  
+After the migration is complete, you switch the brokers to using KRaft and the controllers out of migration mode using the `strimzi.io/kraft: enabled` annotation.
+
+Before starting the migration, verify that your environment can support Kafka in KRaft mode, as there are a number of xref:ref-operator-use-kraft-feature-gate-str[limitations].
+
+WARNING: ZooKeeper to KRaft migration is not suitable for production environments.
```

Comment:
Is this the case? What is the reason?

If nothing else, we should make clear it is not **yet** suitable.

```suggestion
WARNING: ZooKeeper to KRaft migration is not yet suitable for production environments.
```

Reply:
@ppatierno -- do we need this?  (We have the same note in rhel doc)

Reply:
we have a few notes in the guide along this line:  "IMPORTANT
KRaft mode is not ready for production in Apache Kafka or in Strimzi."

Reply:
I think it is ready in Apache Kafka (whether it should be or not, that is a different story). If we don't think it is ready yet in Strimzi, we should make clear it will be. But we also need to understand what is missing and when it will be ready and I think this is currently not clear.

Reply:
From a Strimzi perspective, I consider the migration procedure ready (even for production) but of course the codebase has to be reviewed and approved. From an Apache Kafka perspective, it's defined this way as well but as Jakub says, if it's real or not it's a different story.